crates/sui-core/src/authority_server.rs

```diff
@@ -254,7 +254,6 @@ impl ValidatorService {
         prometheus_registry: Registry,
         rx_reconfigure_consensus: Receiver<ReconfigConsensusMessage>,
     ) -> Result<Self> {
-        let (tx_consensus_to_sui, rx_consensus_to_sui) = channel(1_000);
         let (tx_sui_to_consensus, rx_sui_to_consensus) = channel(1_000);
```

Comment:
Maybe we can rename this to `tx_consensus_listener` and `rx_consensus_listener`? `sui_to_consensus` seems it is only for input into consensus.